cmd/auth-provider-gcp/provider/provider_test.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package provider
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	utilnet "k8s.io/apimachinery/pkg/util/net"
```

Comment:
K8s usually wants the imports of builtins, K8s code and other libraries are 3 separate sections.

Reply:
Gotcha, fixed